templates/mysqlbackup.sh.erb

```diff
@@ -70,18 +70,18 @@ cleanup
 <% if @file_per_database -%>
 mysql --defaults-file=$TMPFILE -s -r -N -e 'SHOW DATABASES' | while read dbname
 do
-  mysqldump --defaults-file=$TMPFILE --opt --flush-logs --single-transaction \
+  mysqldump --defaults-extra-file=$TMPFILE --opt --flush-logs --single-transaction \
```

Comment:
`--defaults-extra-file` present in docs since `5.0` http://dev.mysql.com/doc/refman/5.0/en/mysqldump.html#option_mysqldump_defaults-extra-file. I can't find confirmation for earlier versions quickly.

I faced the problem that mysql backup script can't find socket file. It was caused by mysql built from sources.